cmd/opts.go

```diff
@@ -328,7 +328,8 @@ type UpdateRuntimeConfigOpts struct {
 	VarFlags
 	OpsFlags
 
-	Name string `long:"name" description:"Runtime-Config name (default: '')" default:""`
+	NoRedact bool   `long:"no-redact" description:"Show non-redacted manifest diff"`
```

Comment:
Should add a new test in [`opts_test.go`](https://github.com/cloudfoundry/bosh-cli/blob/35a76411753f85311e82351f1906510f1c494e3a/cmd/opts_test.go#L1089).